### PR TITLE
update capability flags

### PR DESCRIPTION
**Pull request by peterabbott**
Adds the ability to specify the capabilities flag, fix for #104 

Updated README but not bumped the version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -368,6 +368,25 @@ Examples:
...
+### cap_add
+Adds a capability to the running container, for example SYS_PTRACE
+
+````
+cap_add: 
+- SYS_PTRACE
+
+````
+
+### cap_add
```
Comment (mbautin) [inline README.md]: This should be `cap_drop`, right?

File: lib/kitchen/driver/docker.rb [hunk 2]
```diff
@@ -225,6 +227,10 @@ def build_run_command(image_id)
         cmd << " -privileged" if config[:privileged]
         cmd << " -e http_proxy=#{config[:http_proxy]}" if config[:http_proxy]
         cmd << " -e https_proxy=#{config[:https_proxy]}" if config[:https_proxy]
+        if version_above?('1.2.0') 
+          Array(config[:cap_add]).each { |cap| cmd << " --cap-add=#{cap}" } if config[:cap_add]
+          Array(config[:cap_drop]).each { |cap| cmd << " --cap-drop=#{cap}"}  if config[:cap_remove]
```
Comment (mbautin) [inline lib/kitchen/driver/docker.rb]: I think only one of `cap_drop` and `cap_remove` should be used.

**Comment by peterabbott** [comment]
Thanks @mbautin, silly mistakes. Updated and added simple serverspec test for cap-drop.

**Comment by portertech** [comment]
This looks great! :+1:

**Pull request merged**